index.js

```diff
@@ -141,44 +188,61 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
+ i = 0
+    function takeATicketNumber (array){
+      i += 1
+      line.push(i)
+  return `Welcome. you are ticket number ${i}`
+ }
+
```

Comment:
Great work, I would say to define the "i" variable with a let keyword so that it's not global.